package.json

```diff
@@ -191,6 +191,8 @@
   "devDependencies": {
     "@actions/core": "1.10.0",
     "@actions/github": "5.1.1",
+    "@octokit/rest": "^18.0.0",
+    "openai": "^4.24.7",
```

Comment:
small thing but if you just do `npm install openai` it would put the dependency in alphabetical order. Doesn't actually matter, just :eye_twitch:

Reply:
Done, I remember I added these with copy & paste and ran `npm install` which resulted in them not being placed in alphabetical order.